config.py

```diff
@@ -1,7 +1,13 @@
 import logging
 import os
 
-COBOT_ROOT = os.environ.get('COBOT_ROOT', os.getcwd())
+# Deprecate COBOT_ROOT
+BOT_ROOT = os.environ.get('BOT_ROOT', os.getcwd())
+if 'BOT_ROOT' not in os.environ:
```

Comment:
You could change ``if 'BOT_ROOT' not in os.environ`` to ``if 'COBOT_ROOT' in os.environ`` so as to ensure that ``COBOT_ROOT`` is completely removed from the environment and not on only some places.